CHANGELOG.md

```diff
@@ -211,6 +211,7 @@ releasing them early in case they elicit some feedback that requires changes.
 * [Cloud Composer](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/composer/README.md)
 * [Cloud Data Loss Prevention (DLP)](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/dlp/README.md)
 * [Cloud Debugger](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/debugger/README.md)
+* [Filestore](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/filestore/README.md)
```

Comment:
Can you put this in between "Eventarc" and "Game Servers"?

I thought the list looked nicer if we treated "Cloud ___" as starting with a 'C'